app/views/home/index.html.haml

```diff
@@ -84,8 +84,28 @@
         %div.align-items
           %h4.margin-items= t('home.fairness')
           = render Display::InfoTooltipComponent.new(text: "You are seing the average scores for all the public ontologies in AgroPortal. FAIR scores are computed with the O'FAIRe methodology. More details here: https://github.com/agroportal/fairness")
+          = link_to(get_fairness_service_url,id: "fairness-service-url", target: "_blank", "aria-label": t("get_json_version"), title: t("get_json_version")) do
+            %i.fas.fa-code{"aria-hidden": "true", style: "margin-left: 0.5rem" }
+          = link_to("https://github.com/agroportal/fairness", target: "_blank", "aria-label": t("view_fair_scores_definitions"), title: t("view_fair_scores_definitions")) do
+            %i.fas.fa-question-circle{"aria-hidden": "true", style: "margin-left: 0.5rem"}
         %hr.home-section-line/
         .home-card.home-fair-scores
+          %div.fair-scores-badge-section
+            %h2.p-2.badge.badge-secondary.fair-scores-badge-margin
```

Comment:
add here the total score also

Reply:
okay

Reply:
Ah also maybe try to use the primary color for the badges, see if it is better.

Reply:
okey i will see, because i use the same like agroportal (gray)

Reply:
@syphax-bouazzouni , here there is not total score (agroportal)
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/d64ce555-3732-4282-afb5-75725afe044d)